diff --git a/node_modules/react-native-android-widget/android/build.gradle b/node_modules/react-native-android-widget/android/build.gradle
index 2a13816..2727104 100644
--- a/node_modules/react-native-android-widget/android/build.gradle
+++ b/node_modules/react-native-android-widget/android/build.gradle
@@ -69,8 +69,8 @@ android {
   }
 
   compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
+    sourceCompatibility JavaVersion.VERSION_17
+    targetCompatibility JavaVersion.VERSION_17
   }
 
   sourceSets {
